tests/test_boundary_config.py

```diff
@@ -0,0 +1,31 @@
+import unittest
+
+from commons_boundaries.boundary_config import BoundaryConfig
+
+
+class TestBoundaryConfig(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(self):
```

Comment:
If this is a class method, then the first parameter will be the class, not an instance of the class, so it's conventional to call it `cls` rather than `self`.

There's also a small thing that I think I alluded to this before, but not explicitly - I'd tend to use `setUp` unless there's a really good reason that you have to use `setUpClass`, so it's harder to break test isolation (i.e. have the actions of one test method unexpectedly affect another).